src/Database/Database.php

```diff
@@ -1461,8 +1461,13 @@ public function count($table, array $condition = [], array $params = [])
 
 		$row = $this->fetchFirst($sql, $condition);
 
-		// Ensure to always return either a "null" or a numeric value
-		return is_numeric($row['count']) ? (int)$row['count'] : $row['count'];
+		if (empty($row['count'])) {
```

Comment:
This would also be triggered when the `$row['count']` would be 0.

Reply:
So it should be `$row === false`, shouldn't it? :)

Reply:
Or some `if (!isset($row['count'])) {`

Reply:
Yep, this would also work if `$row` is null, for some reason.